lib/BackendClient.js

```diff
@@ -204,33 +201,34 @@ var BackendClient = (function(){
     }
 
     BackendClient.getLiveEntriesForMediaServer = function() {
+        // TODO: Should we transfer logger here as well? Only called from APIQueryAdapter and not used currently -> Gad
        return createSession()
            .then(function() {
                 return getLiveEntries();
             });
     };
 
-    BackendClient.getEntries = function(entryList) {
-        return createSession().
-                then(function () {
-                    return getEntries(entryList);
-                });
+    BackendClient.getEntries = function(entryList, logger) {
+        return createSession(logger)
```

Comment:
actully for createSession it doesn't really matter which logger it is, you can put in the general moduel logger